sound/soc/codecs/rt5682-sdw.c

```diff
@@ -372,7 +372,7 @@ static int rt5682_io_init(struct device *dev, struct sdw_slave *slave)
 	if (rt5682->hw_init)
 		return 0;
 
-	regcache_cache_only(rt5682->regmap, false);
+	regcache_cache_only(rt5682->sdw_regmap, false);
```

Comment:
@bardliao in the suspend routine we do this:

	regcache_cache_only(rt5682->regmap, true);

Should we also do this for sdw_regmap? I am not sure I follow the flows...

Reply:
> @bardliao in the suspend routine we do this:
> 
> ```
> regcache_cache_only(rt5682->regmap, true);
> ```
> 
> Should we also do this for sdw_regmap? I am not sure I follow the flows...

Thanks @plbossart. On the second thought, I think we should set both rt5682->sdw_regmap and rt5682->regmap cache only. Because 1. rt5682_clock_config uses rt5682->sdw_regmap directly. 2. If we set rt5682->sdw_regmap only, rt5682_sdw_read and rt5682_sdw_write won't get/set the right value through rt5682->sdw_regmap.